src/main/java/org/apache/sysds/utils/NativeHelper.java

```diff
@@ -416,4 +416,7 @@ public static native boolean conv2dSparse(int apos, int alen, int[] aix, double[
 	// different tradeoffs. In current implementation, we always use GetPrimitiveArrayCritical as it has proven to be
 	// fastest. We can revisit this decision later and hence I would not recommend removing this method.
 	private static native void setMaxNumThreads(int numThreads);
+
+	//test interface for native binding
```

Comment:
it is not a test inteface anymore, 
please write a full Javadoc definition using:

/** */